lib/abbreviato/truncated_sax_document.rb

```diff
@@ -0,0 +1,194 @@
+require 'nokogiri'
+require 'htmlentities'
+
+class TruncatedSaxDocument < Nokogiri::XML::SAX::Document
+  IGNORABLE_TAGS = %w[html head body].freeze
```

Comment:
but then we don't seem to add them back anywhere nor account for their sizes in the overall truncated string length.